book/src/integrations/oauth2.md

```diff
@@ -70,6 +70,36 @@ anything special for Kanidm (or another provider).
 **Note:** some apps automatically append `/.well-known/openid-configuration` to
 the end of an OIDC Discovery URL, so you may need to omit that.
 
+
+<dl>
+
+<dt>
+
+[Webfinger](https://datatracker.ietf.org/doc/html/rfc7033)
+URL
+
+</dt>
+<dd>
+
+`https://idm.example.com/oauth2/openid/:client_id:/.well-known/webfinger`
+
+The webfinger url is implemented for each openid client, under its specific url, giving full control to the administrator regarding which to use.
+
+To make this webfinger useful it **MUST** be served at the very root under the host (e.g `example.com/.well-known/webfinger`). How that is accomplished is left up to the administrator as kani has no opinion.
```

Comment:
```suggestion
To make this compliant with the standard, it must be made available under the correct [well-known endpoint](https://datatracker.ietf.org/doc/html/rfc7033#section-10.1) (e.g `example.com/.well-known/webfinger`), typically via a reverse proxy or similar. Kanidm doesn't currently provide a mechanism for this URI rewrite.
```

Reply:
I wrote the "must" as `**MUST**` given that is how it is written within the standard (since must has special meaning). but happy to make it boring :P